src/components/styles/baseline.less

```diff
@@ -32,3 +32,15 @@ html {
   right: 0;
   bottom: 0;
 }
+
```

Comment:
This *will not work* in prod.

You need to use the `makeStyles` and `useStyles` helpers from Material UI.

Very brief example is here: https://github.com/MSFS-Mega-Pack/MSFS2020-livery-manager/blob/a93d9188d1e17a5a2126b7b6233833029b36fdb0/src/pages/SplashScreen.js#L9

Reply:
How would we target the table header and last row though?